docs/lang/articles/advanced/differentiable_programming.md

```diff
@@ -169,6 +169,10 @@ for examples on using autodiff-based force evaluation MPM and FEM.
 As mentioned above, `ti.Tape()` can only track a 0D field as the output variable.
 If there're multiple output variables that you want to back-propagate
 gradients to inputs, `kernel.grad()` should be used instead of `ti.Tape()`.
+Different from using `ti.Tape()`, you need to set the `grad` of the output variables themselves to `1` manually
```

Comment:
It's not limited to `1` I guess?

Reply:
Sorry I failed to come up with cases that require non-1 grad for the output variables. Could you please give some examples? :P